source/usr/lib/mcvirt/cluster/cluster.py

```diff
@@ -38,6 +38,11 @@ class RemoteObjectConflict(MCVirtException):
     pass
 
 
+class ClusterNotInitailised(MCVirtException):
```

Comment:
Exceptions class names should end in 'Exception', so this should be called 'ClusterNotInitailisedException'